docs/platforms/apple/common/migration/index.mdx

```diff
@@ -3,7 +3,126 @@ title: Migration Guide
 ---
 
-To upgrade from version 4.x of the SDK to version 8.x of the SDK, you must first migrate from 4.x to 5.x, then 5.x to 6.x, then 6.x to 7.x, and finally from 7.x to 8.x.
+To upgrade from version 4.x of the SDK to version 9.x of the SDK, you must first migrate from 4.x to 5.x, then 5.x to 6.x, then 6.x to 7.x, 7.x to 8.x, and finally from 8.x to 9.x.
+
+## Migrating From 8.x to 9.x
+
+Migrating to 9.x from 8.x includes several breaking changes. We provide this guide to help you update your SDK.
+
+### Changes to Minimum OS Versions and Xcode
+
+We bumped the minimum supported OS versions to:
+- **iOS**: 15.0 (previously 11.0)
+- **macOS**: 10.14 (previously 10.13)
+- **tvOS**: 15.0 (previously 11.0)
+- **visionOS**: 1.0 (unchanged)
+- **watchOS**: 8.0 (previously 4.0)
+
+We now build the precompiled XCFramework with **Xcode 16**. To submit to the App Store, Apple now requires Xcode 16. If you need a precompiled XCFramework built with Xcode 15, continue using Sentry SDK 8.x.x.
+
+Additionally, the Package.swift sets `swift-tools-version` to 6.0.
+
+### Features Enabled by Default
+
+We now enabled the following features by default:
+
+- **Performance V2 is enabled by default**: The app start duration now finishes when the first frame is drawn instead of when the OS posts the `UIWindowDidBecomeVisibleNotification`. We removed the `enablePerformanceV2` option.
+- **Pre-warmed app start tracking**: The SDK now collects pre-warmed app starts by default via `enablePreWarmedAppStartTracing`.
+- <PlatformLink to="/configuration/app-hangs/#app-hangs-v2">App hang tracking V2</PlatformLink>: This is now the default and we removed the option to enable/disable it.
+
+### Disable App Hang Tracking for Extensions
+
+We now automatically disable app hang tracking for: Widgets, Live Activities, Action Extensions, (Siri) Intent Extensions, Share Extensions. These components run in separate processes or sandboxes with different execution characteristics, which can cause false positive app hang reports.
+
+### Structured Logging
+
+We moved structured logging options out of experimental and made them part of the main API. We added log APIs to `Hub` and `Client`, and logs now include a `sentry.replay_id` attribute.
+
+#### HTTP Client Errors
+
+HTTP client errors now mark sessions as errored. This provides better visibility into failed network requests in your release health data.
+
+### Profiling Changes
+
+We removed all deprecated profiling APIs in version 9.0.0. The only supported profiling method is now <PlatformLink to="/profiling/#enable-ui-profiling">UI Profiling</PlatformLink> (also known as continuous V2 profiling), which was introduced in version 8.49.0.
+
+**Removed deprecated profiling options from `SentryOptions`:**
+- `profilesSampleRate` - used for transaction-based profiling (deprecated)
+- `profilesSampler` - used for transaction-based profiling (deprecated)
+- `enableAppLaunchProfiling` - used for launch profiling (deprecated)
```

Comment:
We should also mark these as removed in v9 in `docs/platforms/apple/common/profiling/index.mdx`